db/migrate/20171016115554_add_admin_to_admin_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddAdminToAdminUser < ActiveRecord::Migration[5.1]
+  def change
+    add_column :admin_users, :admin, :boolean, default: false
```

Comment:
Is `admin` the best attribute name? Perhaps `super_admin`? Not sure about this, but `admin` does not feel right when the class name is `AdminUser`